client/packages/common/src/intl/utils/DateUtils.ts

```diff
@@ -31,7 +31,7 @@ const dateInputHandler = (date: Date | string | number): Date => {
 };
 
 export const DateUtils = {
-  getDateOrNull: (date: string | null): Date | null => {
+  getDateOrNull: (date?: string | null): Date | null => {
```

Comment:
Due to a change in the API which is now returning an undefined. easier to change in here rather than the caller